Script/AtomicEditor/ui/frames/MainFrame.ts

```diff
@@ -111,12 +111,18 @@ class MainFrame extends ScriptWidget {
         if (this.menu.handlePopupMenu(target, refid))
             return true;
 
+        if (this.openedMenu && this.openedMenu.id == target.id + " popup") {
+            this.openedMenu.close();
+            this.openedMenu = null;
+            return true;
+        }
+
```

Comment:
I mean the backing TB widget may be released like this, however the UIMenuWindow wrapper will still be held by the openedMenu member, won't be GC'd, and will potentially have a null widget.  I don't think nulling is strictly necessary, but it is something to consider.